### DISCUSSION TITLE
Challenge 1. Cloth mesh.getParticles() returns None

### DISCUSSION THREAD

**Discussion by frankthetank91** [empriselab/RCareWorld#124] [Help - Programming] [answered]
Hello!

I just installed the RCareWorld on Ubuntu 20.04 with CUDA & Nvidia. Although I was able to run the 'example_cloth_attach.py' and other examples. The main test code 'test_dressing.py' does not return the cloth particles. (returns None)
After I did the quick search, I think it is related to the 'Obi' library in RFuniverse. (cloth used in the challenge named 'Obi Cloth' which looks different from the 'example_cloth_attach.py'. 

Would you be able to check this out??

**Comment by YoruCathy** [comment, marked as answer]
https://github.com/empriselab/RCareWorld/blob/phy-robo-care/tests/test_dressing_env.py
Check this test. You need to step after using the get particles api and obtain the data properly.

**Reply by frankthetank91** [reply]
Thank you! I confirmed that 'env.step()' resolved this issue!
